pennylane/devices/task_qubit.py

```diff
@@ -0,0 +1,246 @@
+    as well as class method types. For situations where
+    the explicit class method is needed, we can use this as a
+    decorator, and select the appropriate call dynamically.
+    This is an essential support for accessing backend
+    functionality supports with the proxy `task.qubit`.
+
+    The implementation is based on the Python descriptor guide
+    as mentioned https://stackoverflow.com/questions/28237955/same-name-for-classmethod-and-instancemethod/28238047#28238047
+    """
+
+    def __get__(self, instance, type_):
+        descr_get = super().__get__ if instance is None else self.__func__.__get__
+        return descr_get(instance, type_)
+
+
+# pylint: too-few-public-methods, bad-classmethod-argument
+class TaskQubit(QubitDevice):
+    """Proxy simulator plugin written using Dask.Distributed as a task-distribution scheduling backend.
+
+    **Short name:** ``task.qubit``
+
+    This device provides a pure-state qubit simulator wrapping both ``"default.qubit"`` and ``"lightning.qubit"``,
+    and written to allow batched offloading to a Dask scheduler.
+
+    To use this device, you will need to install dask and dask.distributed:
+
+    .. code-block:: console
+
+        pip install dask distributed
+
+    **Example**
+
+    >>> import pennylane as qml
+    >>> import tensorflow as tf
+    >>> import dask.distributed as dist
+
+    >>> if __name__ == '__main__': # Required for LocalCluster
+    ...     cluster = dist.LocalCluster(n_workers=4, threads_per_worker=1)
+    ...     client = dist.Client(cluster)
+    ...     backend = "default.qubit"
+    ...     dev = qml.device("task.qubit", wires=6, backend=backend)
+    ...     @qml.qnode(dev, cache=False, interface="tf", diff_method="parameter-shift) # caching must be disabled due to proxy interface
+    ...     def circuit(x):
+    ...         qml.RX(x[0], wires=0)
+    ...         qml.RY(x[1], wires=0)
+    ...         qml.RZ(x[2], wires=0)
+    ...         qml.RZ(x[0], wires=1)
+    ...         qml.RX(x[1], wires=1)
+    ...         qml.RY(x[2], wires=1)
+    ...         return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+    ...     weights = tf.Variable(qml.numpy.random.rand(3))
+    ...     def f_submit(weights):
+    ...         with tf.GradientTape() as tape:
+    ...             # Use the circuit to calculate the loss value
+    ...             loss = tf.abs(circuit(weights)-0.5)**2
+    ...         return tape.gradient(loss, weights)
+
+    ...     print(client.submit(f_submit, weights).result())
+    tf.Tensor([0.01776833 0.05199685 0.03689981], shape=(3,), dtype=float64)
+
+    Args:
+        wires (int): The number of wires to initialize the device with.
+        backend (None, str): Indicates the PennyLane device type to use for offloading
+            computation tasks. The TensorFlow and PyTorch interfaces are the preferred types
+            for gradient computations. This is due to existing support in Dask for
+            TF and Torch datatypes.
```

Comment:
Maybe worth noting here that `default.qubit`, one of its subclasses or `lightning.qubit` can be a backend?
```suggestion
            computation tasks. Can be `default.qubit`, one of its subclasses or
            `lightning.qubit`. The TensorFlow and PyTorch interfaces are the preferred
            types for gradient computations. This is due to existing support in Dask for
            TF and Torch datatypes.
```

Reply:
Yep, good point.